Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/FireMonSecurityManager.py

```diff
@@ -0,0 +1,365 @@
+
+            if 'device' in result[i]['pcaResult']:
+                if 'parents' in result[i]['pcaResult']['device']:
+                    del result[i]['pcaResult']['device']['parents']
+                if 'children' in result[i]['pcaResult']['device']:
+                    del result[i]['pcaResult']['device']['children']
+                if 'gpcDirtyDate' in result[i]['pcaResult']['device']:
+                    del result[i]['pcaResult']['device']['gpcDirtyDate']
+                if 'gpcComputeDate' in result[i]['pcaResult']['device']:
+                    del result[i]['pcaResult']['device']['gpcComputeDate']
+                if 'gpcImplementDate' in result[i]['pcaResult']['device']:
+                    del result[i]['pcaResult']['device']['gpcImplementDate']
+                if 'state' in result[i]['pcaResult']['device']:
+                    del result[i]['pcaResult']['device']['state']
+                if 'managedType' in result[i]['pcaResult']['device']:
+                    del result[i]['pcaResult']['device']['managedType']
+                if 'gpcStatus' in result[i]['pcaResult']['device']:
+                    del result[i]['pcaResult']['device']['gpcStatus']
+                if 'updateMemberRuleDoc' in result[i]['pcaResult']['device']:
+                    del result[i]['pcaResult']['device']['updateMemberRuleDoc']
+                if 'devicePack' in result[i]['pcaResult']['device']:
+                    del result[i]['pcaResult']['device']['devicePack']
+
+    return CommandResults(outputs_prefix='FireMonSecurityManager.PCA',
+                          outputs_key_field='pca',
+                          outputs=result,
+                          readable_output=tableToMarkdown(name="FireMon SecurityManager PCA:",
+                                                          t={"pca": result},
+                                                          removeNull=True),
+                          raw_response=list_of_device_changes)
+
+
+def main():
+    username = demisto.params().get('credentials').get('identifier')
+    password = demisto.params().get('credentials').get('password')
+    verify_certificate = not demisto.params().get('insecure', False)
+    base_url = urljoin(demisto.params()['url'])
+    proxy = demisto.params().get('proxy', False)
+    try:
+        client = Client(
+            base_url=base_url,
+            verify=verify_certificate,
+            proxy=proxy,
+            username=username,
+            password=password)
+        if demisto.command() == 'test-module':
+            result = test_module(client)
+            demisto.results(result)
+        elif demisto.command() == 'user-authentication':
+            return_results(authenticate_command(client))
+        elif demisto.command() == 'create-pp-ticket':
+            return_results(create_pp_ticket_command(client, demisto.args()))
+        elif demisto.command() == 'pca':
+            return_results(pca_command(client, demisto.args()))
```

Comment:
Command names should follow conventions, i.e. vendorname-do-something instead of do-something (with the exception of reputation commands)

Please change it, for example - `firemon-pac`, `firemon-create-pp-ticket`

Reply:
See https://xsoar.pan.dev/docs/integrations/code-conventions.